stories/control/legend.stories.tsx

```diff
@@ -0,0 +1,71 @@
+import { Meta } from '@storybook/react';
+import React from 'react';
+
+import { Axis, Legend, LineSeries, Plot } from '../../src';
+
+enum Position {
+  'bottom' = 'bottom',
+  'embedded' = 'embedded',
+  'left' = 'left',
+  'right' = 'right',
+  'top' = 'top',
+}
+
+export default {
+  title: 'API/Legend',
+  component: Legend,
+  argTypes: {
+    position: {
+      control: { type: 'select', options: Position },
+      defaultValue: Position.embedded,
+    },
+  },
+} as Meta;
+
+const data = [
+  {
+    x: 0,
+    y: 10,
+  },
+  {
+    x: 1,
+    y: 12,
+  },
+  {
+    x: 2,
+    y: 14,
+  },
+  {
+    x: 3,
+    y: 16,
+  },
+  {
+    x: 4,
+    y: 18,
+  },
+];
+
+interface LegendControlProps {
+  position: Position;
+}
+
+export function HeadingControl(props: LegendControlProps) {
+  return (
+    <Plot
+      width={900}
+      height={540}
+      viewportStyle={{ stroke: 'black' }}
+      margin={{
+        bottom: 100,
+        left: 40,
+        top: 40,
+        right: 40,
```

Comment:
Please increase the margins for this story, so that we can see the legend when the position is left, right, top, ...